package.json

```
@@ -11,7 +11,7 @@
     "deploy:prod": "fly deploy -a delphi-tg-bots-prod -c fly.prod.toml",
     "logs": "fly logs",
     "logs:prod": "fly logs -a delphi-tg-bots-prod",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "mocha"
```

when I run `npm test` on this branch it says no tests found:

```
❯ npm test

> delphi-reads-tg-bot@1.0.0 test
> mocha

Error: No test files found
```

I think you need to update this to:

```suggestion
    "test": "NODE_OPTIONS='--import tsx' mocha test/*.spec.ts"
```